### PR TITLE
Expose Blazor components dpi

### LINKED ISSUE #1831: [FEATURE] Expose Blazor components dpi field
**Is your feature request related to a problem? Please describe.**

It would be useful to expose the dpi field in the `SKCanvasView` and `SKGLView` components.

**Describe the solution you'd like**

Add a public property on both components to expose the private field.

**Describe alternatives you've considered**

None.

### PR DESCRIPTION
**Pull request by beto-rodriguez**
**Description of Change**

It would be useful to expose the dpi field in the `SKCanvasView` and `SKGLView` components.

**Bugs Fixed**

fixes #1831

- Related to issue #

**API Changes**

Added on `SKCanvasView` : 

- public double DPI => dpi; 

Added on `SKGLView` : 

- public double DPI => dpi; 

**Behavioral Changes**

**PR Checklist**

- [ ] Has tests (if omitted, state reason in description)
- [ ] Rebased on top of main at time of PR
- [ ] Changes adhere to coding standard
- [ ] Updated documentation

### CODE REVIEW

File: source/SkiaSharp.Views.Blazor/SkiaSharp.Views.Blazor/SKCanvasView.razor.cs [hunk 1]
```diff
@@ -60,6 +60,8 @@ public bool IgnorePixelScaling
 		[Parameter(CaptureUnmatchedValues = true)]
 		public IReadOnlyDictionary<string, object>? AdditionalAttributes { get; set; }
 
+		public double DPI => dpi;
```
Comment (robloo) [inline source/SkiaSharp.Views.Blazor/SkiaSharp.Views.Blazor/SKCanvasView.razor.cs:63]: `DPI` -> `Dpi`

> The PascalCasing convention, used for all identifiers except parameter names, capitalizes the first character of each word (including acronyms over two letters in length), as shown in the following examples:

https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions

File: source/SkiaSharp.Views.Blazor/SkiaSharp.Views.Blazor/SKGLView.razor.cs [hunk 2]
```diff
@@ -67,6 +67,8 @@ public bool IgnorePixelScaling
 		[Parameter(CaptureUnmatchedValues = true)]
 		public IReadOnlyDictionary<string, object>? AdditionalAttributes { get; set; }
 
+		public double DPI => dpi;
```
Comment (robloo) [inline source/SkiaSharp.Views.Blazor/SkiaSharp.Views.Blazor/SKGLView.razor.cs:70]: Same as above `Dpi`

**Comment by beto-rodriguez** [comment]
The previous commit applies the rename @robloo

**Comment by quomatz** [comment]
Please merge into main finally :(

**Review by mattleibow — APPROVED** [review]

**Comment by beto-rodriguez** [comment]
@dotnet-policy-service agree

**Comment by mattleibow** [comment]
Thanks for staying with us :) I know it has been a long time.

**Pull request merged**